### ISSUE TITLE
Mimic Support for URDF & Equality Support for MJCF

### ISSUE THREAD

**Issue by COMoER** [Genesis-Embodied-AI/Genesis#129]
Thanks for releasing such an awesome simulator for robotics researchers. For simulation of the gripper or dexterous hand, the complex linkage structures are often simplified as underactuated joints, which are defined as the mimic attribute or equality attribute in URDF or MJCF respectively. Is there any support for mimic joints in Genesis? I would appreciate it if you could reply.

**Comment by Richal13Yu** [comment]
Same question here

**Comment by RaneWouters** [comment]
Same question here.

**Comment by zhouxian** [comment]
Yes this could be a helpful feature and should be relatively simple to implement

**Comment by YilingQiao** [comment]
We now support MJCF's equality constraint and will enhance its compatibility with ball joints in a future PR.
https://github.com/Genesis-Embodied-AI/Genesis/pull/636
